### DISCUSSION TITLE
Turbo Prune Optimization

### DISCUSSION THREAD

**Discussion by yovanoc** [vercel/turborepo#3026] [Ideas]
Why the turbo prune command with --docker flag doesn't take only dist folders and package.json files  to reduce docker image size ?

**Comment by anthonyshew** [comment]
You would need the source code files to build after you created a pruned lockfile. If you want to optimize your image to include even less, you can create a build stage after you have your outputs like the one seen here: https://turbo.build/repo/docs/handbook/deploying-with-docker#example
